docs/index.rst

```diff
@@ -0,0 +1,40 @@
+Oppia User Guide
+============================================
+
+.. meta::
+   :description lang=en: A comprehensive user guide for using the Oppia platform
+
+
+Oppia is an open source learning platform with the goal of providing engaging content to learners. Anyone can create and share interactive lessons (explorations) with the aim of recreating a one-on-one tutoring scenario.    
+
+This interactivity is what makes Oppia unique. Many of the educational tools on the Internet provide a unidirectional flow of information which makes it hard to create engaging, feedback-rich experiences for learners. However, Oppia comes with several built-in interactions and responses that make it possible to personalize feedback to a learner's answer. Over time, explorations can be improved as more learners engage with the content. 
+
+Oppia's mission is to provide high-quality, accessible learning experiences to everyone around the world.
+
+To begin, we suggest getting  familiar with the :ref:`key terms used in Oppia <keyconcepts>` first.
+
+
+:ref:`Lesson Creators - start here<create>`
```

Comment:
Should we add space before `<` here and elsewhere similar to line 14? (or let's change line 14.)

Reply:
I think any convention is fine, as long as it's consistent. Probably the one in line 14 is more readable?

Reply:
I have added a space now in all files for consistency. Done